src/Template/Contributions/activity_timeline.ctp

```diff
@@ -43,81 +43,131 @@ $selectedMonth = format(
     __('{month} {year}'),
     array('month' => $monthName, 'year' => $year)
 );
- 
-$maxWidth = 400;
-$maxTotal = 0;
 
-foreach ($stats as $stat) {
-    if ($stat->sentences > $maxTotal) {
-        $maxTotal = $stat->sentences;
-    }
+$maxWidth = 400;
+if (!empty($stats)) {
+    $maxTotal = max(array_column($stats, 'total'));
 }
 ?>
 
 <div id="annexe_content">
-    <?php 
+    <?php
     echo $this->element(
-        'calendar', 
+        'calendar',
         array(
             'currentYear' => $year,
             'currentMonth' => $month
         )
-    ); 
+    );
     ?>
 </div>
 
 <div id="main_content">
     <div class="section md-whiteframe-1dp">
-    <h2><?php echo $selectedMonth; ?></h2>
+    <h2><?php echo __('Activity timeline') . ' - ' . $selectedMonth; ?></h2>
```

Comment:
Could you also fix this string the same way? We can then get rid of this `$selectedMonth` variable entirely.